cfy_manager/components/rabbitmq/rabbitmq.py

```diff
@@ -409,7 +409,9 @@ def _generate_rabbitmq_certs(self):
             # anything consuming this value will get the path to the cert
             # that will allow them to trust the broker.
             config[RABBITMQ]['ca_path'] = config[RABBITMQ]['cert_path']
-        if len(config[RABBITMQ]['cluster_members']) > 1:
+
+        cluster_members = config[RABBITMQ]['cluster_members']
```

Comment:
What's the purpose of this change?

Reply:
I changed it :)
I think it was because line 422 was too long, but now it is fine.